src/screens/OnBoardingScreen.tsx

```diff
@@ -139,8 +137,9 @@ const Select = (props: {
   return (
     <View style={{ paddingVertical: 8 }}>
       <Typography variant="h6">{props.title}</Typography>
-      {props.values.map((value) => (
+      {props.values.map((value, index) => (
         <View
+          key={index}
```

Comment:
Try to not use index as a key.
Video: https://egghead.io/lessons/react-use-the-key-prop-when-rendering-a-list-with-react
Documentation: https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
That is true. So we are setting index as key, which is what react does as default, just to get the error message away?

Reply:
OK, I'll let this one slip 😅 as long as we switch it out with a proper key when we move over to dynamic data 💪